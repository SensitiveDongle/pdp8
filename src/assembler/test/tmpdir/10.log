Undefined symbol program used at line 4
*** 1 errors in assembly
