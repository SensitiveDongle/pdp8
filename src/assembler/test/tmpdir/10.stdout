  0x000: 0xE80    1: 	CLA
  0x001: 0xE0A    2: 	RTR
  0x002: 0xF02    3: 	HLT
                  4: END program
